packages/patternlab/styleguide/source/_patterns/03-organisms/by-author/inline-links.twig

```diff
@@ -0,0 +1,15 @@
+{% if inlineLinks.links %}
+  <section class="ma__inline-links {{ (inlineLinks.background)?"ma__inline-links--"~inlineLinks.background:"" }}">
+    <nav class="ma__inline-links__container" role="navigation" {% if inlineLinks.ariaLabel %} aria-label="{{ inlineLinks.ariaLabel }} {% endif %}">
```

Comment:
```suggestion
    <nav class="ma__inline-links__container" {% if inlineLinks.ariaLabel %} aria-label="{{ inlineLinks.ariaLabel }} {% endif %}">
```
role navigation is not necessary for `nav`

Reply:
I wonder semantically, if it makes more sense to use <ul> and <li> for this element

Reply:
Removed the "role" attribute since it's not necessary in this context.